.github/workflows/python-threatexchange-ci.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8]
```

Comment:
If we are removing dataclass usage, why NOT test against 3.6?

Reply:
Ohh. I had thought this was testing for hmalib. I see now for pyte. reverting